src/index.ts

```diff
@@ -20,11 +22,28 @@ declare module 'fluture' {
   }
 }
 
-export const fluture: Monad<URI> & Bifunctor<URI> = {
+interface Next<T> {
+  done: false
+  value: T
+}
+
+interface Done<T> {
+  done: true
+  value: T
+}
```

Comment:
I will release a new minor version of Fluture in the coming days which will expose these interfaces.